### PR TITLE
Added a new api to get logs

### PR DESCRIPTION
**Pull request by mariarivera30**
Added a new api for auditlogs and added support for created_from and created_to for all api

### CODE REVIEW

File: pybossa/api/auditlog.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf8 -*-
+# This file is part of PYBOSSA.
+#
+# Copyright (C) 2015 Scifabric LTD.
```
Comment (dchhabda) [inline pybossa/api/auditlog.py]: this can be updated to latest period

File: pybossa/auth/auditlog.py [hunk 2]
```diff
@@ -35,10 +35,14 @@ def _create(self, user, auditlog, project_id=None):
         return False
 
     def _read(self, user, auditlog=None, project_id=None):
-        if user.is_anonymous or (auditlog is None and project_id is None):
+        if user.is_anonymous or (auditlog is None and project_id is None and not user.admin):
```
Comment (dchhabda) [inline pybossa/auth/auditlog.py]: if we move #41 <if user.admin return True> as first check in this func, check of not user.admin can be removed/simplified.

File: pybossa/api/auditlog.py (hunk 1 above)
Comment (mariarivera30) [reply pybossa/api/auditlog.py]: What do you mean by the latest period? 2019?

Comment (dchhabda) [reply pybossa/api/auditlog.py]: yes

File: pybossa/auth/auditlog.py (hunk 2 above)
Comment (mariarivera30) [reply pybossa/auth/auditlog.py]: If i move #41 we will check for user.admin before checking user.is_anonymous. Is that ok ?

**Pull request merged**